data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -57,7 +57,17 @@
     "time": 5000,
     "autolearn": true,
     "qualities": [ { "id": "SCREW", "level": 1 } ],
-    "components": [ [ [ "battery", 125 ] ] ]
+    "components": [
+      [
+        [ "light_minus_battery_cell", 20 ],
+        [ "light_battery_cell", 10 ],
+        [ "light_plus_battery_cell", 5 ],
+        [ "medium_battery_cell", 2 ],
+        [ "medium_plus_battery_cell", 2 ],
+        [ "heavy_battery_cell", 1 ],
+        [ "heavy_plus_battery_cell", 1 ]
```

Comment:
So these battery cells are all acid-based? Not Li-Ion, not Li-Poly?

Reply:
Um. I dunno. They're rechargeable, but I wanted to keep as much compatibility with existing recipes as possible.

Reply:
I'd remove the recipe altogether. It always looked weird to me. But that's only my opinion.

Reply:
As far as I know, there's only two acid recipes and I'm not sure if the other is possible without acid rain. I'm keeping this for now so people can craft acid at all, but later I'll add some non-rechargeable batteries that are acid based, and move the recipes over.